src/bindings/jna/libelektra5j/src/main/java/org/libelektra/Elektra.java

```diff
@@ -146,22 +146,17 @@ int elektraGOptsContractFromStrings (Pointer contract, long argsSize, String arg
 
 	Pointer ksCut (Pointer ks, Pointer cutpoint);
 
+	// TODO #3137 Also elektraKsPopAtCursor should replace the current ksPop. See also #3189.
 	Pointer ksPop (Pointer ks);
 
+	@Deprecated(forRemoval = true)
```

Comment:
```suggestion
	@Deprecated
```

It seems [some build servers](https://github.com/ElektraInitiative/libelektra/pull/3706/checks?check_run_id=2125713117) still use Java 8. Until that changes, we can't use the `forRemoval` parameter.

Reply:
Thank you for the information. This is important to know since i of course was planning on using JDK 11 syntax. Is there a plan to update the build servers?

Reply:
Since everything is based on Dockerfiles or some sort of YAML files anyway, updating to JDK 11 (or even newer) should be that hard. Maybe @mpranj or @robaerd know of any specific reasons for using JDK 8, otherwise I think you can just update everything.

Reply:
Can i open an issue to increase the jdk version to 11 while still buildin for target 8?

Reply:
Sure, that works too